### PR TITLE
unit tests

### PR DESCRIPTION
**Pull request by IsharaSandeepanie**

### CODE REVIEW

File: components/org.wso2.carbon.identity.remotefetch.core/src/test/java/org/wso2/carbon/identity/remotefetch/core/dao/impl/RemoteFetchConfigurationDAOImplTest.java [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+        DataSource dataSource = mock(DataSource.class);
+        mockStatic(JdbcUtils.class);
+        when(JdbcUtils.getNewTemplate()).thenReturn(new JdbcTemplate(dataSource));
+        try (Connection connection = DAOTestUtils.getConnection(DB_NAME)) {
+            Connection spy = DAOTestUtils.spyConnection(connection);
+            when(dataSource.getConnection()).thenReturn(spy);
+            remoteFetchConfigurationDAO.createRemoteFetchConfiguration(createRemoteFetch());
+        }
+    }
+
```
Comment (ashensw) [inline components/org.wso2.carbon.identity.remotefetch.core/src/test/java/org/wso2/carbon/identity/remotefetch/core/dao/impl/RemoteFetchConfigurationDAOImplTest.java]: Remove unnecessary newlines.

Comment (IsharaSandeepanie) [reply components/org.wso2.carbon.identity.remotefetch.core/src/test/java/org/wso2/carbon/identity/remotefetch/core/dao/impl/RemoteFetchConfigurationDAOImplTest.java]: [fixed](https://github.com/wso2-extensions/identity-fetch-remote/pull/13/commits/aeee8586ae82876e6ba95c9a889fa6312129898f)

File: components/org.wso2.carbon.identity.remotefetch.core.ui/src/test/resources/testng.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (ashensw) [inline components/org.wso2.carbon.identity.remotefetch.core.ui/src/test/resources/testng.xml]: Update the license header

File: components/org.wso2.carbon.identity.remotefetch.core/src/test/resources/testng.xml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (ashensw) [inline components/org.wso2.carbon.identity.remotefetch.core/src/test/resources/testng.xml]: Update the license header.

Comment (IsharaSandeepanie) [reply components/org.wso2.carbon.identity.remotefetch.core/src/test/resources/testng.xml]: [Fixed](https://github.com/wso2-extensions/identity-fetch-remote/pull/13/commits/21eee2a3e6dc27743cc668f4f3aaf303ece32859)

File: components/org.wso2.carbon.identity.remotefetch.core.ui/src/test/resources/testng.xml (hunk 2 above)
Comment (IsharaSandeepanie) [reply components/org.wso2.carbon.identity.remotefetch.core.ui/src/test/resources/testng.xml]: [fixed](https://github.com/wso2-extensions/identity-fetch-remote/pull/13/commits/21eee2a3e6dc27743cc668f4f3aaf303ece32859)

**Pull request merged**